dali/pipeline/pipeline.cc

```diff
@@ -236,6 +250,31 @@ void Pipeline::AddOperator(OpSpec spec, const std::string& inst_name) {
   this->op_specs_to_serialize_.push_back(true);
 }
 
+inline void Pipeline::AddSplittedNvJpegDecoder(OpSpec &spec, const std::string& inst_name) {
+  spec.set_name("nvJPEGDecoderCPUStage");
+  spec.SetArg("device", "cpu");
+
+  auto& op_output = spec.MutableOutput(0);
+  string op_output_name = op_output.first;
+
+  op_output.first = "nvJPEGCPUOutput0";
```

Comment:
I would mangle it with the name of the operator or something, otherwise if user uses two of those operators you would get clashing tensor names in the pipeline/graph and everything will fail.

Reply:
Was mainly for debug and forgot to update it, done